src/app/autorun/autorun.component.ts

```diff
@@ -111,7 +111,7 @@ export class AutorunComponent implements OnInit {
     }
     else {
       this.requestBody.inputFtpDetails = null;
-      this.requestBody.inputFileDetails.fileExtension = 'xlsx';  //*** */
+      this.requestBody.inputFileDetails.fileExtension = this.inputFile.name.split(".").pop();  //*** */
```

Comment:
@utraksh - can u remove those comments //*** */, if its not making any sense?

Reply:
Done. @naveenkumarkandasamy .

Reply:
@utraksh please remove the comments and push